scenario27/jmeter/27-pre-scenario-steps.sh

```diff
@@ -18,6 +18,20 @@ serverHost=$serverHost
 serverPort=$serverPort
 tomcatHost=$tomcatHost
 tomcatPort=$tomcatPort
+
+serverPort=$serverPort
```

Comment:
Remove the duplicate variables

Reply:
Thanks for pointing out I removed the duplicate values